### PR TITLE
Create css.lua

### PR DESCRIPTION
**Pull request by elefantapan**
Idk if it can get better without modifying Godot source code

( I don't have time for it )

yall can do it without me then

### CODE REVIEW

**Comment by QuinceTart10** [comment]
Nice work, haven't tested it yet, but I see that you haven't added any comments.

**Comment by face-hh** [comment]
@elefantapan could you please add the comments so I can merge this?

**Review by RealPacket — CHANGES_REQUESTED** [review]
Needs some cleanup
You'll need to add comments, as it will need
I'll try to add some comments *later*.
Also,
I never finished the last review which was supposed to fix the repetition,
that's why it took a long time

File: Lua/Plugins/css.lua [hunk 1]
```diff
@@ -0,0 +1,866 @@
...
+highlight("word-spacing", "function")
+highlight("word-wrap", "function")
+highlight("writing-mode", "function")
+highlight("z-index", "function")
+
+--- Special Characters
+highlight("{", "symbol")
+highlight("#", "symbol")
+highlight(".", "symbol")
+
```
Comment (ItsTato) [inline Lua/Plugins/css.lua:491]:
```suggestion
highlight("}", "symbol")

```

File: Lua/Plugins/css.lua [hunk 2]
```diff
@@ -0,0 +1,866 @@
...
+highlight("h2", "reserved")
+highlight("h3", "reserved")
+highlight("h4", "reserved")
+highlight("h5", "reserved")
+highlight("h6", "reserved")
+highlight("head", "reserved")
+highlight("header", "reserved")
+highlight("hgroup", "reserved")
+highlight("hr", "reserved")
+highlight("html", "reserved")
```
Comment (ItsTato) [inline Lua/Plugins/css.lua:61]:
```suggestion

```

File: Lua/Plugins/css.lua [hunk 3]
```diff
@@ -0,0 +1,866 @@
...
+highlight("form", "reserved")
+highlight("frame", "reserved")
+highlight("frameset", "reserved")
+highlight("h1", "reserved")
+highlight("h2", "reserved")
+highlight("h3", "reserved")
+highlight("h4", "reserved")
+highlight("h5", "reserved")
+highlight("h6", "reserved")
+highlight("head", "reserved")
```
Comment (ItsTato) [inline Lua/Plugins/css.lua:57]:
```suggestion

```

File: Lua/Plugins/css.lua [hunk 4]
```diff
@@ -0,0 +1,866 @@
...
+highlight("picture", "reserved")
+highlight("pre", "reserved")
+highlight("progress", "reserved")
+highlight("q", "reserved")
+highlight("rp", "reserved")
+highlight("rt", "reserved")
+highlight("ruby", "reserved")
+highlight("s", "reserved")
+highlight("samp", "reserved")
+highlight("script", "reserved")
```
Comment (ItsTato) [inline Lua/Plugins/css.lua:97]:
```suggestion

```

File: Lua/Plugins/css.lua [hunk 5]
```diff
@@ -0,0 +1,866 @@
...
+highlight("table", "reserved")
+highlight("tbody", "reserved")
+highlight("td", "reserved")
+highlight("template", "reserved")
+highlight("textarea", "reserved")
+highlight("tfoot", "reserved")
+highlight("th", "reserved")
+highlight("thead", "reserved")
+highlight("time", "reserved")
+highlight("title", "reserved")
```
Comment (ItsTato) [inline Lua/Plugins/css.lua:119]:
```suggestion

```

File: Lua/Plugins/css.lua [hunk 6]
```diff
@@ -0,0 +1,866 @@
...
+highlight("kbd", "reserved")
+highlight("label", "reserved")
+highlight("legend", "reserved")
+highlight("li", "reserved")
+highlight("link", "reserved")
+highlight("main", "reserved")
+highlight("map", "reserved")
+highlight("mark", "reserved")
+highlight("menu", "reserved")
+highlight("meta", "reserved")
```
Comment (ItsTato) [inline Lua/Plugins/css.lua:76]:
```suggestion

```

**Review by ItsTato — CHANGES_REQUESTED** [review]

**Pull request merged**